### PR TITLE
Centralize getting os version as drawBot.macOSVersion.macOSVersion

### PR DESCRIPTION
**Pull request by justvanrossum**
- Adapt other code to it
- Fixed a wrong version comparison in pdfContext
- Use relative imports here and there

### CODE REVIEW

File: drawBot/macOSVersion.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from distutils.version import StrictVersion
+import platform
+
+# It is safe to compare osVersion to strings, as StrictVersion casts strings
+# to StrictVersion instances upon compare.
+macOSVersion = StrictVersion(platform.mac_ver()[0])
```
Comment (typemytype) [inline drawBot/macOSVersion.py]: Check if the platform is not None and use a fallback instead

See `AttributeError: 'StrictVersion' object has no attribute 'version'` on read the docs

Comment (justvanrossum) [reply drawBot/macOSVersion.py]: Yes, I think I found a solution.

Comment (justvanrossum) [reply drawBot/macOSVersion.py]: Does RTD build PR branches? If not, we may have to merge, and continue on master if it still doesn't work.

Comment (justvanrossum) [reply drawBot/macOSVersion.py]: Ah right, it builds "stable" and "latest". I'm fairly confident this PR will fix the RTD build, though, so I think we should merge (once Travis is done :) ).

**Pull request merged**